src/database/model/realtimeDataModel.ts

```diff
@@ -0,0 +1,25 @@
+import {Model} from '@nozbe/watermelondb';
+import {field, date} from '@nozbe/watermelondb/decorators';
+
+export default class RealtimeDataModel extends Model {
+  static table = 'realtime_data';
+
+  @field('distance') distance!: number;
+  @field('elapsed_time') elapsed_time!: number;
+  @field('speed') speed!: number;
+  @field('latitude') latitude!: number;
+  @field('longitude') longitude!: number;
+  @field('altitude') altitude!: number;
+  @field('heading') heading!: number;
+  @field('heart_rate') heart_rate!: number;
+  @field('instant_power') instant_power!: number;
+  @field('three_sec_power') three_sec_power!: number;
+  @field('ten_sec_power') ten_sec_power!: number;
+  @field('cadence') cadence!: number;
+  @field('is_bluetooth') is_bluetooth!: boolean;
+  @field('is_wifi') is_wifi!: boolean;
+  @field('is_power') is_power!: boolean;
+  @field('is_power') is_heart_rate!: boolean;
+  @field('is_power') is_cadence!: boolean;
+  @date('created_at') createdAt!: number;
```

Comment:
Should we add `updated_at` ? It feels like it may be useful.

Reply:
I'm not sure if it matters yet, but have you thought about how we might validate whether data in this table is stale? For instance if a sensor disconnects or something?

Reply:
I think all tables automatically get updated_at and id in watermelon. I haven't given much thought to how we check the freshness of the data, but will have a think about it.

Reply:
Looking at their docs I think you have to set it like `created_at`. They also suggest adding a `readonly` decorator. But let's look at that when we need it.

https://watermelondb.dev/Advanced/CreateUpdateTracking.html